pages/college_predictor.js

```diff
@@ -36,10 +45,18 @@ const CollegePredictor = () => {
                 );
 
                 // Sort the filteredData in ascending order of college rank
+                // for same college (with same rank), sort with opening rank
                 filteredData.sort((a, b) => {
                     const rankA = a["College Rank"];
                     const rankB = b["College Rank"];
-                    return rankA - rankB;
+
+                    if (rankA !== rankB) {
```

Comment:
Why do we need a customized sort? If we're just sorting by the `"College Rank"` column - a simple sort won't work?

Reply:
you only wrote it :p let me check whats the best way to sort object data in js